src/test/java/cucumber/steps/SetComposedEndpointStep.java

```diff
@@ -0,0 +1,38 @@
+package cucumber.steps;
+
+import clickup.utils.ScenarioContext;
+import core.api.ApiRequest;
+import core.api.ApiRequestBuilder;
+import core.api.ApiResponse;
+import io.cucumber.java.en.Given;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+import static clickup.utils.getPathParamsNames.getPathParamsFromEndpoint;
+
+public class SetComposedEndpointStep {
+    private ApiRequestBuilder apiRequestBuilder;
+    private ApiRequest apiRequest;
+    private ApiResponse apiResponse;
+    private SoftAssert softAssert;
+    ScenarioContext scenarioContext = ScenarioContext.getInstance();
+
+    public SetComposedEndpointStep(ApiRequestBuilder apiRequestBuilder, ApiResponse apiResponse,
+                                   SoftAssert softAssert) {
+        this.apiRequestBuilder = apiRequestBuilder;
+        this.apiResponse = apiResponse;
+        this.softAssert = softAssert;
+    }
+
+    @Given("^I set the (.*) composed endpoint (.*)$")
```

Comment:
This step exists in ApiSteps

Reply:
This step is for composed endpoints, because the endpoint used in this step has two params instead of one